db/seeds.rb

```diff
@@ -19,7 +18,9 @@
 
 # Create a site
 puts "- adding default site"
-w = Website.create!(:title => 'Development', :subdomain => 'development')
+w = Website.new(:title => 'Development', :subdomain => 'development')
+w.users << user
```

Comment:
Great work @KevinSjoberg.
I think this will raise an `ActiveRecord::AssociationTypeMismatch` if `user` becomes `nil`. It is possible due to the code att row 7-12.

If you are able to verify my concern and fix that issue it would be great and I would be happy to merge your pull request.